lib/fswatcher/event_types_fen.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2017 The Syncthing Authors.
```

Comment:
They don't really overlap, I could unify them using runtime.GOOS in init(). Both (buildtags and runtime.GOOS) are used in the rest of the code, so I don't see whether there is a specific reason to use one or the other. Should I unify it with runtime?